### PR TITLE
BA/Cards

### PR DESCRIPTION
**Pull request by pnegahdar**

### CODE REVIEW

**Comment by matthewfl** [comment]
I am assuming that this should not be in a pull request yet, as you haven't implement the functionality of the delete function

File: test/test.js [hunk 1]
```diff
@@ -172,6 +172,16 @@ series([
             next("api.Cards.create2");
         });
     },
+    function (next) {
+        api.Cards.delete(myCard.uri, function (err) {
```
Comment (matthewfl) [inline test/test.js]: is myCard used in a later test?

Comment (matthewfl) [inline test/test.js]: if you are changing the name to unstore then why is the delete

File: lib/nbalanced/card.js [hunk 2]
```diff
@@ -171,3 +171,18 @@ card.prototype.invalidate = function(card_uri, callback) {
     }, callback);
 
 }; // invalidate
+
+
+card.prototype.unstore = card.prototype.delete = function (item_uri, callback) {
```
Comment (matthewfl) [inline lib/nbalanced/card.js:181]: Have you noticed how every function has a nice comment block above it?

File: lib/nbalanced/bankAccount.js [hunk 3]
```diff
@@ -316,18 +316,14 @@ bankAccount.prototype.credits = function (credits_uri, callback) {
...
-    // Need to make it something like this /v1/bank_accounts/:bank_account_id
-    var hackyDeleteUri = item_uri.replace(self._marketplace_uri, "/v1");
-
     utility.apiCall({
         marketplace_uri: self._marketplace_uri,
         secret: self._secret,
-        uri: hackyDeleteUri,
+        uri: item_uri,
         method: "DELETE",
         override_uri: true
```
Comment (matthewfl) [inline lib/nbalanced/bankAccount.js]: I believe that having taking out the `hackyDeleteUri` that this is no longer needed here, plz run the tests and check

**Comment by mjallday** [comment]
still missing docstring

**Pull request merged**